src/mongoose_scram.erl

```diff
@@ -101,23 +101,25 @@ mask(Key, Data) ->
     <<C:KeySize>>.
 
 enabled(Host) ->
-    ejabberd_auth:get_opt(Host, password_format) == scram.
+    PasswordFormat = ejabberd_auth:get_opt(Host, password_format),
+    ScramSha = proplists:get_value(scram, [PasswordFormat]),
+    PasswordFormat == scram orelse PasswordFormat == {scram, ScramSha}.
```

Comment:
It took me a while to understand what happens here. To my understanding this is equivalent of the following:
```suggestion
    case ejabberd_auth:get_opt(Host, password_format) of
        scram -> true;
        {scram, _ScramSha} -> true;
        _ -> false
    end.
```